docs/01-overview/02-main-areas/eventing/evnt-01-overview.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Overview
+---
+
+Eventing in Kyma can be used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CRD](../technical-reference/subtab_customresources/eventing-event-subscription.md).
```

Comment:
"can it be used" or is it used? Are there other purposes than sending/receiving events?

Reply:
You're right, "is used" would be better 🙂